tests/foreman/api/test_lifecycleenvironment.py

```diff
@@ -24,204 +24,203 @@
 
-            1. Create an organization.
-            2. Create a lifecycle environment belonging to the organization.
-            3. Search for lifecycle environments in the organization.
+    :expectedresults: Lifecycle environment is not updated and
+        corresponding error is raised
 
-        :expectedresults: Only "Library" and the lifecycle environment just
-            created are in the search results.
+    :CaseImportance: Low
+    """
+    with pytest.raises(HTTPError):
+        module_lce.name = new_name
+        module_lce.update(['name'])
+    lce = module_lce.read()
+    assert lce.name != new_name
 
-        :CaseLevel: Integration
-        """
-        org = entities.Organization().create()
-        lc_env = entities.LifecycleEnvironment(organization=org).create()
-        lc_envs = lc_env.search({'organization'})
-        self.assertEqual(len(lc_envs), 2)
-        self.assertEqual({lc_env_.name for lc_env_ in lc_envs}, {'Library', lc_env.name})
 
-    @tier2
-    @pytest.mark.stubbed('Implement once BZ1348727 is fixed')
-    def test_positive_create_environment_after_host_register(self):
-        """Verify that no error is thrown when creating an evironment after
-        registering a host to Library.
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+@tier1
+@upgrade
+def test_positive_delete(lce, name):
+    """Create lifecycle environment and then delete it.
 
-        :id: ceedf88d-1ad1-47ff-aab1-04587a8121ee
+    :id: cd5a97ca-c1e8-41c7-8d6b-f908916b24e1
 
-        :BZ: 1348727
+    :expectedresults: Lifecycle environment is deleted successfully
 
-        :Setup:
-            1. Create an organization.
-            2. Create a new content host.
-            3. Register the content host to the Library environment.
+    :CaseImportance: Critical
+    """
+    lce.delete()
+    with pytest.raises(HTTPError):
+        entities.LifecycleEnvironment(id=lce.id).read()
 
-        :Steps: Create a new environment.
 
-        :expectedresults: The environment is created without any errors.
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
+@tier2
+def test_positive_search_in_org(module_org, name):
+    """Search for a lifecycle environment and specify an org ID.
 
-        :CaseLevel: Integration
+    :id: 110e4777-c374-4365-b676-b1db4552fe51
 
-        :CaseAutomation: notautomated
-        """
+    :Steps:
+
+        1. Create an organization.
+        2. Create a lifecycle environment belonging to the organization.
+        3. Search for lifecycle environments in the organization.
+
+    :expectedresults: Only "Library" and the lifecycle environment just
+        created are in the search results.
+
+    :CaseLevel: Integration
+    """
+    lc_env = entities.LifecycleEnvironment(organization=module_org).create()
+    lc_envs = lc_env.search({'organization'})
+    assert len(lc_envs) == 2
+    assert {lc_env_.name for lc_env_ in lc_envs}, {u'Library', lc_env.name}
```

Comment:
Does that work?

module_org is module scoped, so all your tests here will share single organization. Unless this is very first test executed, I would expect this organization to have multiple lifecycle environments.

I'm afraid you need to create organization explicitly for this test, either inside function or using function-scoped fixture.

Reply:
This does work, but seems to fail for the stand alone jenkins run.  I'll see what if I can fix this.